lib/types.ts

```diff
@@ -1,8 +1,11 @@
 'use strict';
 // Thanks to https://github.com/kbajalc/parquets for some of the code.
-import * as BSON from "bson"
-import { PrimitiveType, OriginalType, ParquetType, FieldDefinition, ParquetField } from "./declare"
+import { PrimitiveType, OriginalType, ParquetType, FieldDefinition, ParquetField } from "./declare";
 import { Options } from "./codec/types";
+import type { Document as BsonDocument } from "bson";
+// BSON uses top level awaits, so use require for now
+const bsonSerialize = require('bson').serialize;
+const bsonDeserialize = require('bson').deserialize;
```

Comment:
Really annoying and only impacts esbuild for the browser, but not too bad.

Why the double require? It was clearer than the named version and keeps the linter away from trying to reformat it.